atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance_div = document.querySelector("div#checking_balance");
+var savingsBalance_div = document.querySelector("div#savings_balance");
+checkingBalance = 0;
+savingsBalance = 0;
+console.log(checkingBalance);
 
 var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+  checkingBalance += parseInt(amount);
+  checkingBalance_div.innerHTML = "$" + checkingBalance;
+  if(checkingBalance >= 0){
+    console.log("hi");
+    $("div#checking_balance").removeClass("zero");
+  }
+
```

Comment:
Thank you. That actually makes a lot of sense. I"ll try doing that for the project I'm working on right now. I think it'll be better than "hi" lol